src/pages/profile/profileModule.ts

```diff
@@ -0,0 +1,10 @@
+import Elysia from "elysia";
+import { authModule, requireAuth } from "../../auth/authModule";
+import { Profile } from "./Profile";
+import { ProfileSettings } from "./ProfileSettings";
+
+export const profileModule = new Elysia()
+  .use(authModule)
```

Comment:
Even though the auth module is installed in index.tsx, we still want to install it here too so that the `beforeHandle` used below get proper linting. We will rely on Elysia's pluging deduplication as discussed [here](https://github.com/adsilence/Gamertime/pull/17/files#r1460717065) to not have it actually installed multiple times.